tests/test_perf.py

```diff
@@ -151,3 +158,19 @@ def test_perf_thread_comm_is_process_comm(
         #   pative 1925947 [010] 987095.272656: PERF_RECORD_COMM: pative:1925904/1925947
         # we take the exec comm for all threads so we remain with the first, "native".
         _assert_comm_in_profile(profiler, application_pid, True)
+
+@pytest.mark.parametrize("runtime", ["native_thread_comm"])
+@pytest.mark.parametrize("perf_mode", ["fp"])
+@pytest.mark.parametrize("insert_dso_name", [False, True])
+@pytest.mark.parametrize("in_container", [True])
+def test_dso_name_in_perf_profile(
+    system_profiler: SystemProfiler,
+    application_pid: int,
+    insert_dso_name: bool,
+    # tmp_path: Path,
+    # perf_mode: str
+) -> None:
+    with system_profiler as profiler:
+        collapsed = snapshot_pid_profile(profiler, application_pid).stacks
+        assert is_function_in_collapsed("recursive", collapsed)
+        assert insert_dso_name == is_function_in_collapsed("recursive [/native]", collapsed)
```

Comment:
I thought about it, since for Python we use `( )` to notate the filename there, let's keep it consistent and use `( )` for native filenames as well. So Instead of adding `[dso]`, let's add `(dso)` and keep it consistent. The same we will do for other runtimes when we add it.

Reply:
Just to be sure: the final format for all profiler backends would be `<function> ' ' '(' <dso-name> ')'`, right?

Reply:
Yes. And if no function name - `'(' <dso-name> ')'`

Reply:
:+1